### ISSUE TITLE
disbanding the accessibility meetings until after a new governance structure is adopted 

### ISSUE THREAD

**Issue by tonyfast** [jupyter/accessibility#146]
during our last minute we discussed what we could do to maintain the accessibility group. this was a beautiful discussion because everyone was interested in keeping the meetings going voluntarily; everyone wants accessibility to improve that much. unfortunately, i don't think its possible for us to improve jupyter accessibility on purely voluntary efforts. accessibility needs resources that we have at the moment so this issue proposes that we disband the accessibility meetings until new governance is formed.

the jupyter accessibility has been hosting bi-weekly [meetings] for nearly four years, since @isabela-pf  started them in 2020. in that time, we have succeeded in raising community awareness and advocacy, but we've failed to produce an accessible version of intertactive computing for assistive technology users. we've had strong impacts on the developer community, but we haven't improved conditions for assistive technology users. we've had funding and made minor progress. we've failed to address professional jupyter [audits]. there are reasons for this and we should assess. unfortunately, there is zero [funding] for accessibility and that means we have little ability to improve conditions. i think we should save our energy and spirits.

jupyter has been working hard in recent years to mature its identity, it seems like the recent [linux foundation](https://github.com/jupyter/governance/issues/204) inquiry might provide the growth needed for a project of this size. i suggest that we disband our accessibility meetings and work asynchronously until jupyter establishes its new identity. when that time comes we can advocate for more accessibility resources, but until these changes happen i think we will be expending borrowed resources that might be better saved up until we understand the future of jupyter better.

if this effort does disbands and regroups in new governance structure then accessibility, if governance is serious about accessibility, should play a more central role in DEIA, physical/digital community events and all of the user interfaces under the jupyter umbrella.

cc: @ohrely @gabalafou 

[funding]: https://jupyter-accessibility.readthedocs.io/en/latest/funding/index.html
[audits]: https://jupyter-accessibility.readthedocs.io/en/latest/audits/index.html
[meetings]: https://jupyter-accessibility.readthedocs.io/en/latest/community/meeting-minutes/jupyterlab-accessibility-meetings/2020/index.html

**Comment by gabalafou** [comment]
Thanks, Tony. I think what you've written makes a lot of sense. 

My one concern with disbanding the meeting is whether the it can serve in helping to organize for some of things you wrote about above, with respect to accessibility's place in the possible transition to Linux Foundation. Or is what you're suggesting that we take a hiatus until after the matter of the Linux Foundation has been settled and then, potentially, to regroup?

**Comment by tonyfast** [comment]
i am assuming if accessibility were critical to the LF transition someone would have talked to us. we have the steering council reps and accessibility that can act on any foundation related things async. to me, the best course of action is to wait til LF is formed and advocate for accessibility/security more then.

**Comment by gabalafou** [comment]
So, since today's call was a bit in question for me, I decided to err on the side having the meeting. I'm glad that I did because a couple folks from UC Berkley showed up to ask if a student could show-and-tell a basic accessibility checker JupyterLab extension they developed. 

I guess that I'm of the mind that it's good to keep the conversation going. I'm in a position to continue to host the meeting every two weeks for anyone who wants to show up. I feel you that it's not enough—that we need more to get to the ultimate goal—but I guess I'm having a hard time making the case to myself that the call does more harm than good, although I'm open to that possibility, too.

**Comment by isabela-pf** [comment]
Thanks for bringing this up, @tonyfast. I've been feeling more and more unsteady with this group since it was formalized as a part of Jupyter's governance, so it's validating to hear someone else's concerns too. I'm certainly open to a more explicit pause of some or all of accessibility group routines (like the meetings) if others are interested considering that I have not been able to be much involved of late.

While it's tangential to the issue of meetings, I will say I am uncertain about the status of the group in governance overall. This has been years of effort and, as @tonyfast mentioned, a lot of care from a lot of people. I remember why there was a push to bring accessibility work into a place where it was recognized formally by Jupyter governance (ie. an accessibility-minded vote on things that needed voting, accountability to keep the effort ongoing, etc.). I cannot honestly say that I think those motivations have panned out in reality. And as digital accessibility advocacy continues to advance in legal and tech spaces over the years and Jupyter does not, I am afraid that subproject is inching towards pinning accountability for inaccessibility on the individuals involved in the subproject and not Project Jupyter as a whole organization. I don't think we are there yet, but I do think it's worth asking who the systems we set up serve and recognizing that they have not served the members of the subproject in any of the ways we were hoping upon joining. If this resonates with anyone, I'm open to starting a separate discussion with more details. Let me know.

**Comment by tonyfast** [comment]
> I'm certainly open to a more explicit pause of some or all of accessibility group routines (like the meetings)

@isabela-pf it's good to hear that this also makes sense to someone else; especially someone whose leadership has been really important to this effort.

> While it's tangential to the issue of meetings, I will say I am uncertain about the status of the group in governance overall.

this was the origin of the thought to regroup the accessibility meetings -  jupyter lacks both consistent governance and vision for accessibility, which resulted in the struggle in launching our [what does accessibility mean to jupyter](https://docs.google.com/document/d/1m5-fiqWVdlqOuKjp4auakG0MMaTii7JdCVyjwKU7uQU/edit?usp=sharing) workshop. getting alignment on shared values and goals would be critical for considering onboarding additional collaborators[new subprojects like jupyter book](https://github.com/jupyter/enhancement-proposals/issues/122). 

> I cannot honestly say that I think those motivations have panned out in reality. 

this is hard to come to terms with, but i agree. so many people have done so much good work supporting the community, but there has also been a lot of burnout and attrition. while we have had affected users eager to join, we haven't been able to support them fully - to @gabalafou's mention of the last meeting - our efforts can definitely do some good, but we lack the ability to affect systemic change.

> And as digital accessibility advocacy continues to advance in legal and tech spaces over the years and Jupyter does not, I am afraid that subproject is inching towards pinning accountability for inaccessibility on the individuals involved in the subproject and not Project Jupyter as a whole organization. 

the position of accessibility in jupyter is not central, but it needs to be. accessibility has the same dynamics as security - they should both be core features of a product, considered and designed for  - side efforts can't address these issues in a sustainable way. i have similar concerns to @isabela-pf about having accountability and responsibility for accessibility in jupyter that isn't matched by our resourcing or agency  - with that in mind, i'm stepping back from this work and if our cohort considers this collectively, we may have a better chance of getting clarity on governance and support.

**Comment by jasongrout** [comment]
To move the conversation forward, what would the accessibility subproject do if it had access to substantial sustained resources, say in the tens of thousands of dollars a year (think maybe $20,000 - $50,000)? What would be the most valuable things to do in the next 2 years?

If we move to LF, we're poised to have much more resources as a project. It would do well for us to collectively think ahead about how we might use such resources for the most valuable next steps in various areas of the project, especially valuable cross-project efforts that have had very little resources up to this point like accessibility.

**Comment by tonyfast** [comment]
> To move the conversation forward, what would the accessibility subproject do if it had access to substantial sustained resources

thanks for those questions @jasongrout . i'll take some time to answer them in a few days in #147. i'd like to avoid adding more scope to this issue as it is constrained by to whether we should continue our bi-weekly group meetings.

**Comment by gabalafou** [comment]
Want to follow up on this.

Tomorrow there will be a call because at the previous call, I committed to hosting the call for a [student demo](https://discourse.jupyter.org/t/accessibility-demo-tomorrow/25988).

But the general sentiment seems that we should disband the call for now, and I agree with the arguments brought forward, so I'm on board with that. I will solicit feedback from the council on the mailing list. If no one objects or steps forward in favor of keeping the meeting going, then I will open some PRs to update the places were we advertise the meeting on the web to let people know it's not happening anymore.

**Comment by trallard** [comment]
I have been swamped, so I am only following up on this discussion. 

I sympathise with the sentiment in this conversation. We have hit a point where we are running on volunteer time and with minimal support/backing from Jupyter governance (read EC and how decision-making and alignment are working under the current governance of Jupyter). Thus, the team as a whole has run out of steam. We have reached the limit of what we can do without stronger and continuous support from the EC to ensure accessibility becomes a core consideration across all projects within Jupyter.

I am +1 on **pausing** the calls and regrouping at a later time. 
It also _might be_ that the transition to the LF breathes new winds and life into Jupiter's governance as a whole (though being realistic, this is still in very early stages and seeing the impact of this move will take a non-trivial amount of time).

Though I would like to propose setting a date at which we will regroup and assess direction and goals, otherwise I am worried that an indefinite pause will mean we lose whatever momentum and /or interest we have built and continue to keep.

**Comment by afshin** [comment]
I'll attend tomorrow (13 June). It's still on the calendar and I wouldn't want to learn that a new contributor tried to show up and nobody was there.

Please come if you are free and inclined.
